contracts/exchange-forwarder/contracts/src/MixinExchangeWrapper.sol

```diff
@@ -401,4 +483,15 @@ contract MixinExchangeWrapper is
             return false;
         }
     }
+
+    /// @dev Checks whether an order is a v2 order.
+    /// @param order Order struct containing order specifications.
+    /// @return True if the order's `makerFeeAssetData` is set to the v2 order id.
+    function _isV2Order(LibOrder.Order memory order)
+        internal
+        pure
+        returns (bool)
+    {
+        return order.makerFeeAssetData.readBytes4(0) == EXCHANGE_V2_ORDER_ID;
```

Comment:
This will revert if the order has no maker fee asset.

Reply:
Good catch!